nhost/compose/config.go

```diff
@@ -519,6 +523,8 @@ func (c Config) hasuraServiceEnvs() env {
 		"HASURA_GRAPHQL_ADMIN_SECRET":              util.ADMIN_SECRET,
 		"NHOST_ADMIN_SECRET":                       util.ADMIN_SECRET,
 		"NHOST_BACKEND_URL":                        c.envValueNhostBackendUrl(),
+		"NHOST_SUBDOMAIN":                          "localhost",
```

Comment:
I am not sure why we need it (I am adding it to where we also have `NHOST_BACKEND_URL`) in storage.